### DISCUSSION TITLE
FLUTTER WEB APPS CAN'T DEPLOY ON VERCEL; BUILDS ON LOCAL

### DISCUSSION THREAD

**Discussion by Akaiso** [vercel/community#7158] [Help]
### Page to Investigate

https://vercel.com/new/i-bugs-projects/import/hidden?s=https%3A%2F%2Fgithub.com%2FAkaiso%2Fdosganar-e_commerce&hasTrialAvailable=1&showOptionalTeamCreation=false&project-name=dosganar-e-commerce&framework=other&totalProjects=1&remainingProjects=1&deploymentIds=dpl_CcU6PrkcJcb961VKVaGQg5k5wD8T

### Steps to Reproduce

``<html>
<body>

Compiling lib/main.dart for the Web...
--
08:24:00.034 | Warning: In index.html:37: Local variable for "serviceWorkerVersion" is deprecated. Use "{{flutter_service_worker_version}}" template token instead.
08:24:00.035 | Warning: In index.html:46: "FlutterLoader.loadEntrypoint" is deprecated. Use "FlutterLoader.load" instead.
08:24:16.571 | Target dart2js failed: ProcessException: Process exited abnormally with exit code 1:
08:24:16.571 | lib/view/HomePage.dart:4:1:
08:24:16.571 | Error: 'CarouselController' is imported from both 'package:carousel_slider/carousel_controller.dart' and 'package:flutter/src/material/carousel.dart'.
08:24:16.571 | import 'package:flutter/material.dart';
08:24:16.571 | ^^^^^^^^^^^^^^^^^^
08:24:16.571 | ../.pub-cache/hosted/pub.dev/carousel_slider-4.2.1/lib/carousel_slider.dart:9:1:
08:24:16.571 | Error: 'CarouselController' is imported from both 'package:carousel_slider/carousel_controller.dart' and 'package:flutter/src/material/carousel.dart'.
08:24:16.571 | import 'carousel_controller.dart';
08:24:16.572 | ^^^^^^^^^^^^^^^^^^
08:24:16.572 | lib/view/HomePage.dart:221:43:
08:24:16.572 | Error: 'CarouselController' is imported from both 'package:carousel_slider/carousel_controller.dart' and 'package:flutter/src/material/carousel.dart'.
08:24:16.572 | CarouselController carouselController = CarouselController();
08:24:16.572 | ^^^^^^^^^^^^^^^^^^
08:24:16.572 | ../.pub-cache/hosted/pub.dev/carousel_slider-4.2.1/lib/carousel_slider.dart:48:15:
08:24:16.572 | Error: 'CarouselController' is imported from both 'package:carousel_slider/carousel_controller.dart' and 'package:flutter/src/material/carousel.dart'.
08:24:16.572 | : CarouselController() as CarouselControllerImpl,
08:24:16.573 | ^^^^^^^^^^^^^^^^^^
08:24:16.573 | ../.pub-cache/hosted/pub.dev/carousel_slider-4.2.1/lib/carousel_slider.dart:62:15:
08:24:16.573 | Error: 'CarouselController' is imported from both 'package:carousel_slider/carousel_controller.dart' and 'package:flutter/src/material/carousel.dart'.
08:24:16.573 | : CarouselController() as CarouselControllerImpl,
08:24:16.573 | ^^^^^^^^^^^^^^^^^^
08:24:16.573 | Error: Compilation failed.
08:24:16.573 | Command: /vercel/path0/flutter/bin/cache/dart-sdk/bin/dart --disable-dart-dev /vercel/path0/flutter/bin/cache/dart-sdk/bin/snapshots/dart2js.dart.snapshot --platform-binaries=/vercel/path0/flutter/bin/cache/flutter_web_sdk/kernel --invoker=flutter_tool -Ddart.vm.product=true -DFLUTTER_WEB_CANVASKIT_URL=https://www.gstatic.com/flutter-canvaskit/1cdbebee19012cee6cfaf67a434e3a6fcbc4682b/ -DFLUTTER_WEB_AUTO_DETECT=false -DFLUTTER_WEB_USE_SKIA=true --native-null-assertions --no-source-maps -o /vercel/path0/.dart_tool/flutter_build/5c65c02bdc2bde5289f1a1125e2e1b4d/app.dill --packages=.dart_tool/package_config.json --cfe-only /vercel/path0/.dart_tool/flutter_build/5c65c02bdc2bde5289f1a1125e2e1b4d/main.dart

</body>
</html>

**Comment by virokalu** [comment]
I got the same error 
Error: 'CarouselController' is imported from both 'package:carousel_slider/carousel_controller.dart' and 'package:flutter/src/material/carousel.dart'.
After force updating the flutter to 3.3 or something.
Then I switch to the stable channel of flutter and use flutter 3.2 or something
`flutter channel stable`

Btw I don't know whether this is a correct way or not

**Reply by mangugeng** [reply]
me too

[a comment by an automated account is not shown]
